src/ui/common/src/stories/DataListTable.stories.tsx

```diff
@@ -0,0 +1,232 @@
+import { Box, Typography } from '@mui/material';
+import React from 'react';
+
+import CheckItem, {
+  CheckPreview,
+} from '../components/pages/workflows/components/CheckItem';
+import ExecutionStatusLink from '../components/pages/workflows/components/ExecutionStatusLink';
+import MetricItem, {
+  MetricPreview,
+} from '../components/pages/workflows/components/MetricItem';
+import PaginatedSearchTable, {
+  PaginatedSearchTableData,
+} from '../components/tables/PaginatedSearchTable';
+import { CheckLevel } from '../utils/operators';
+import ExecutionStatus from '../utils/shared';
+
+export const DataListTable: React.FC = () => {
+  const checkPreviews: CheckPreview[] = [
```

Comment:
delete these?

Reply:
or is this the intended way to use story book?

Reply:
this is a mock table for storybook.